lib/util.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+##################################################
+# Validates if the GITHUB_TOKEN is set in the
+# .ashrc file.  Prompts the user to set one if it
+# hasn't been set yet.
+#
+# @returns: 0 if there is a Github token set at
+#               the end of this function call
+#           1 if there is *not* a Github token set
+#               at the end of this function call
+##################################################
+Github_validate_token(){
+    # Checking if variable exists
+    if [[ -z "$GITHUB_TOKEN" ]]; then
+        local response
+        Logger__alert "GITHUB_TOKEN is not set in the .ashrc file"
+        Logger__prompt "Would you like to set one now? (y/n): "; read response
+
+        # If user wants to input their token
+        if [[ "$response" = "y" ]]; then
+            local github_token
+            Logger__prompt "Enter Github Token: "; read github_token
```

Comment:
would be nice to add an additional prompt that linked to `https://github.com/settings/tokens` to tell your end-users where to go to get this token